apps/testrunner/queue.py

```diff
@@ -0,0 +1,105 @@
+import json
+import time
+
+import requests
+import sys
+
+from .logger import logger
+from .runner import SimpleTest
+
+
+class TaskRepo(object):
```

Comment:
TaskRepo and FakeTaskRepo should be somehow related to each other. Maybe FakeTaskRepo should be derived from TaskRepo?